### PR TITLE
Refresh topbeat.yml with changes from libbeat.yml.

### PR DESCRIPTION
**Pull request by andrewkroh**
Closes elastic/libbeat#160

### CODE REVIEW

File: etc/topbeat.yml [hunk 1]
```diff
@@ -9,50 +9,181 @@ input:
...
   elasticsearch:
+
+    # Set to true to enable elasticsearch output
     enabled: true
+
+    # Array of hosts to connect to.
+    # Scheme and port can be left out and will be set to the default (http and 9200)
+    # In case you specify and additional path, the scheme is required: http://localhost:9200/path
+    # IPv6 addresses should always be defined as: https://[2001:db8::1]:9200
     hosts: ["localhost:9200"]
```
Comment (ruflin) [inline etc/topbeat.yml]: I suggest we always use http://localhost:9200 as default

**Comment by ruflin** [comment]
LGTM

**Comment by andrewkroh** [comment]
@monicasarbu I updated commented out line to be `# index: "topbeat"`.

**Comment by monicasarbu** [comment]
@andrewkroh Can you please replace beatname in here:
 
# Optional index name. The default is "beatname" and generates
-  # path: where to save the files		+    # [beatname-]YYYY.MM.DD keys.

File: etc/topbeat.yml (hunk 1 above)
Comment (monicasarbu) [inline etc/topbeat.yml]: I think it's not clear description. I would say here that you can configure the Elasticsearch host as an IP followed by the port or an URL. Some people would prefer one way or the other.  Ipv6 address need to be defined as [2001:db8::1]:9200.

**Comment by andrewkroh** [comment]
@monicasarbu I fixed up the comments. I also fixed a few other places that referred to packetbeat.

**Pull request merged**